tools_docker/README.md

```diff
@@ -146,13 +145,64 @@ Exemple to use Tracim with ElasticSearch-ingest: (you need to create your elasti
 
 ⚠ After execute one of these command, Tracim will be reachable on your system on port 8080.
 
+#### Running with gocryptfs encryption
+
+This need the new specific Debian_New_Uwsgi docker (see build section). This is an experimental
+docker image, the new feature from this docker will maybe be merged to the standard docker or removed.
+Exemple with basic instance of Tracim (local usage with webdav and caldav) with encrypted storage:
+
+for this exemple,
+you first need to write password you want in `~/tracim/secret/password.txt` file.
+
+```bash
+echo "password" > ~/tracim/secret/password.txt
+```
+
+Note: this is just an example, we suggest you to write password with a text editor
+instead in order to not store the password in the bash history.
+
+```bash
+docker run \
+       --device /dev/fuse
+       --cap-add SYS_ADMIN
+       --security-opt apparmor:unconfined
+       -e DATABASE_TYPE=sqlite \
+       -e ENABLE_GOCRYPTFS_ENCRYPTION=1 \
+       -e GOCRYPTFS_PREVIEW_STORAGE_DIR=/var/tracim/previews \
+       -e TRACIM_PREVIEW_CACHE_DIR=/media/previews \
+       -e GOCRYPTFS_UPLOADED_FILES_STORAGE_DIR=/var/tracim/uploaded_files \
+       -e TRACIM_DEPOT_STORAGE_DIR=/media/uploaded_files \
+       -e GOCRYPTFS_PASSWORD_PATH=/var/secret/password.txt \
+       -p 8081:80 \
```

Comment:
Why not use the standard port we have define for tracim. default value `-p 8080:80`.

Reply:
right.

Reply:
done.